crates/testing/src/view_generator.rs

```diff
@@ -289,6 +315,20 @@ impl TestViewGenerator {
             tracing::error!("Cannot attach transactions to the genesis view.");
         }
     }
+
+    pub fn add_view_sync_finalize(
```

Comment:
I think we need a similar function that makes it so a quorum proposal for a view does not reference a QC for a prior view

Reply:
The proposals can be overridden from within the generated values, so we can just change whatever we want. Is this sufficient?

Reply:
Done.